### PR TITLE
[Images] Remove ml-models image

### PR DESCRIPTION
**Pull request by alonmr**
ml-models is no longer maintained. removing it from the build action and makefile.
it was used in demos and contains dependencies overhead for that.
demos will use their own images insead.

### CODE REVIEW

File: mlrun/api/utils/builder.py [hunk 1]
```diff
@@ -624,6 +624,7 @@ def build_runtime(
         runtime.status.state = mlrun.common.schemas.FunctionState.ready
         return True
     if build.base_image:
+        # TODO: ml-models was removed in 1.5.0. remove it from here in 1.7.0.
```
Comment (liranbg) [inline mlrun/api/utils/builder.py:627]: Can't we do something about it now? Or we need to keep it in order to make old mlrun versions based clients?

File: mlrun/runtimes/mpijob/v1alpha1.py [hunk 2]
```diff
@@ -25,6 +25,7 @@
 from mlrun.utils import get_in, update_in
 
 
+# TODO: v1alpha1 is not used anymore, remove this class
```
Comment (liranbg) [inline mlrun/runtimes/mpijob/v1alpha1.py:28]: Add version and deprecation warning

**Review by liranbg — COMMENTED** [review]
Minor

File: mlrun/api/utils/builder.py (hunk 1 above)
Comment (alonmr) [reply mlrun/api/utils/builder.py:627]: the latter

**Review by liranbg — APPROVED** [review]

**Pull request merged**